### PR TITLE
Ads.f117/unifydependecies

### PR DESCRIPTION
**Pull request by adsf117**
@fanky10 in this file now we will have all dependencies and with groovy we can continue adding or update

### CODE REVIEW

File: buildsystem/dependencies.gradle [hunk 1]
```diff
@@ -0,0 +1,59 @@
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+ext {
```
Comment (fanky10) [inline buildsystem/dependencies.gradle]: I don't see the Kotlin version definition

Comment (adsf117) [reply buildsystem/dependencies.gradle]: done!

File: app/build.gradle [hunk 2]
```diff
@@ -29,21 +30,27 @@ dependencies {
...
+
+    implementation appDependencies.picasso,
+            appDependencies.recyclerView,
+            appDependencies.constraintLayout
+
+    testImplementation generalTestDependencies.mockito,
+            generalTestDependencies.junit
+
     // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```
Comment (fanky10) [inline app/build.gradle]: this one should be moved to ext as well, just in case we want to add different kotlin

Comment (adsf117) [reply app/build.gradle]: done!

**Pull request merged**